app/assets/javascripts/spree/backend/solidus_pay_tomorrow.js

```diff
@@ -1,2 +1,4 @@
 // Placeholder manifest file.
-// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/backend/all.js'
\ No newline at end of file
+// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/backend/all.js'
+
+//= require solidus_pay_tomorrow
```

Comment:
I tested it in the backend and PayTomorrow gives me an error when I visit it.

http://localhost:3000/admin/orders/R104148028/payments/new

Can you confirm you are getting the same error?

Reply:
Yes, getting the same error. Fixed in https://github.com/nebulab/solidus_pay_tomorrow/pull/19/commits/e768d8b347b0580bca06d5f76f0956d5195033a0

Reply:
What do you think about just disabling it in the backend? 
Maybe we can just show a message like:

`PayTomorrow can be performed only on frontend`

Reply:
Yea, I am thinking the same.

Reply:
Done